docs/language/functions.rst

```diff
@@ -353,6 +353,35 @@ values. Here is an example of an overloaded function:
 In the function foo, abs() is called with an ``int64`` so the second implementation
 of the function abs() is called.
 
+.. note::
+  The substrate target runtime requires function names to be unique.
+  Overloaded function names will be mangled in the ABI.
+  The function name will be concatenated with all of its argument types, separated by underscores.
+  Additionally, these rules apply:
+  - Struct types are represented by their field types (preceded with an extra underscore).
+  - Enum types are represented as their underlying uint8 type.
+  - Array types are recognizable by having ``Array`` appended. 
+  - Fixed size arrays will additionally have their length appended as well.
```

Comment:
This list is not rendered correctly, see
![image](https://user-images.githubusercontent.com/816900/194055036-5f902642-7f61-4693-b492-e23b54158a25.png)